src/test_l1_tvSettings.c

```diff
@@ -85,58 +85,24 @@ volatile bool callbackflag = false;
 #define MAX_DIMMING_MODES        20
```

Comment:
Remove line no 83 "//TODO: During cleanup identif proper values and remove the macros."

Reply:
Removed the comment.